subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformValuesInjectionIntegrationTest.groovy

```diff
@@ -125,9 +125,8 @@ class ArtifactTransformValuesInjectionIntegrationTest extends AbstractDependency
         fails(":a:resolve")
 
         then:
-        failure.assertHasDescription('A problem occurred evaluating root project')
         // TODO wolfs: We should report the user declared type
-        failure.assertHasCause('Some problems were found with the configuration of MakeGreen$Inject.')
+        failure.assertHasDescription('Some problems were found with the configuration of MakeGreen$Inject.')
```

Comment:
We should probably add a contextual exception to explain what this is

Reply:
What context can we provide? The project which registered the transform? The from/to attributes? Anything else?